src/org/usfirst/frc/team2335/robot/commands/Lift.java

```diff
@@ -0,0 +1,41 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class Lift extends Command
+{
+    public Lift() 
+    {
+        requires(Robot.climb);
+    }
+
+    
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute() 
+    {
+    	//Starts the climb process
+    	Robot.climb.startClimb();
+    }
+    
+    protected boolean isFinished() 
+    {
+        return false;
```

Comment:
OK, thanks for the explanation. As a future TODO we may want to consider writing up a basic guide to our code framework explaining details like this.